### PR TITLE
app-text/master-pdf-editor: correction to 5.4.38 ebuild

### PR DESCRIPTION
**Pull request by jpataias**
Since master-pdf-editor is not compatible with Qt 5.14 the ebuild was corrected so that the bundled libraries are used.

### CODE REVIEW

**Comment by jpataias** [comment]
Closes: https://bugs.gentoo.org/715288

File: app-text/master-pdf-editor/master-pdf-editor-5.4.38-r1.ebuild [hunk 1]
```diff
@@ -14,26 +14,26 @@ SLOT="0"
...
-	>=dev-qt/qtcore-5.9.8:5
-	>=dev-qt/qtgui-5.9.8:5
-	>=dev-qt/qtnetwork-5.9.8:5
-	>=dev-qt/qtprintsupport-5.9.8:5
-	>=dev-qt/qtsvg-5.9.8:5
-	>=media-gfx/sane-backends-1.0
-"
+RDEPEND=">=media-gfx/sane-backends-1.0"
 
 QA_PREBUILT="/opt/${PN}/masterpdfeditor5"
```
Comment (bearsh) [inline app-text/master-pdf-editor/master-pdf-editor-5.4.38-r1.ebuild:19]: to silent QA warnings:
```
QA_PREBUILT="/opt/${PN}/masterpdfeditor5
        /opt/${PN}/lib/*.so*
        /opt/${PN}/iconengines/*.so*
        /opt/${PN}/platformthemes/*.so*
        /opt/${PN}/printsupport/*.so*
        /opt/${PN}/platforms/*.so*
        /opt/${PN}/imageformats/*.so*
"
```

Comment (jpataias) [reply app-text/master-pdf-editor/master-pdf-editor-5.4.38-r1.ebuild]: I was wondering about that, thanks.

The required changes were added.

**Comment by juippis** [comment]
@a17r could you take a look? You might have some ideas.

**Pull request closed without merging**